cloud/docker/docker.py

```diff
@@ -838,6 +848,24 @@ def ensure_capability(self, capability, fail=True):
                     '.'.join(map(str, self.docker_py_versioninfo)),
                     api_version))
 
+    def get_environment(self, env, env_file):
+        """
+        If environment files are combined with explicit environment variables, the explicit environment variables will override the KW from the env file.
+        """
+        final_env = {}
+
+        if env_file:
+            parsed_env_file = docker.utils.parse_env_file(env_file)
```

Comment:
This'll fail with a nasty exception if your docker-py is too old. The way to fail more gracefully (with a more user-friendly error message) would be to do an explicit docker-py version check; see [`_cap_ver_req`](https://github.com/gesellix/ansible-modules-core/blob/add_env_file_support_v2/cloud/docker/docker.py#L655-L673) and [`ensure_capability()`](https://github.com/gesellix/ansible-modules-core/blob/add_env_file_support_v2/cloud/docker/docker.py#L822-L849).